src/pages/layer-2.js

```diff
@@ -0,0 +1,618 @@
+import InfoBanner from "../components/InfoBanner"
+import Link from "../components/Link"
+import PageHero from "../components/PageHero"
+import PageMetadata from "../components/PageMetadata"
+import Pill from "../components/Pill"
+import ProductCard from "../components/ProductCard"
+import { CardGrid, Content, Page } from "../components/SharedStyledComponents"
+
+// Utils
+import { translateMessageId } from "../utils/translations"
+
+// Styles
+const GappedPage = styled(Page)`
+  gap: 4rem;
+  @media (max-width: ${({ theme }) => theme.breakpoints.l}) {
+    gap: 3rem;
+  }
+  * {
+    scroll-margin-top: 5.5rem;
+  }
+`
+
+const HeroContainer = styled.div`
+  background: ${({ theme }) => theme.colors.runNodeGradient};
+  width: 100%;
+`
+
+const Hero = styled(PageHero)`
+  padding-bottom: 2rem;
+`
+
+const Flex50 = styled.div`
+  flex: 50%;
+  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
+    flex: 100%;
+  }
+`
+
+const TwoColumnContent = styled.div`
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 2rem;
+  gap: 2rem;
+  @media (max-width: ${({ theme }) => theme.breakpoints.l}) {
+    flex-direction: column;
+    align-items: flex-start;
+    margin-left: 0rem;
+    margin-right: 0rem;
+  }
+`
+
+const InfoGrid = styled(CardGrid)`
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(min(100%, 340px), 1fr));
+  gap: 1rem 2rem;
+  & > div {
+    height: fit-content;
+    &:hover {
+      transition: 0.1s;
+      transform: scale(1.01);
+      img {
+        transition: 0.1s;
+        transform: scale(1.1);
+      }
+    }
+  }
+`
+
+const StyledCardGrid = styled(CardGrid)`
+  margin-bottom: 4rem;
+  margin-top: 4rem;
+`
+
+const Layer2Page = ({ data }) => {
+  const intl = useIntl()
+  const heroContent = {
+    title: "Layer 2",
+    header: "Layer 2",
+    subtitle: "Smart solutions to scale Ethereum",
+    image: getImage(data.ethBlocks),
+    alt: "test",
+    buttons: [
+      {
+        content: "Use Layer 2",
+        pathId: "use-layer-2",
+      },
+      {
+        content: "Wait, what's Ethereum?",
+        pathId: "/",
+        isSecondary: "isSecondary",
+      },
+    ],
+  }
+
+  const layer2Cards = [
+    {
+      image: privacyGlyph,
+      title: "Inclusivity",
+      preview: "Lower fees and better user experience",
+      body: [
+        "By combining multiple transactions into a single transaction on layer 1, transaction fees are massively reduced, making Ethereum more accessible for all.",
+      ],
+      alt: "test",
+    },
+    {
+      image: privacyGlyph,
+      title: "Expand Use Cases",
+      preview: "Increase the scope of projects",
+      body: [
+        "With higher transactions per second, cheaper fees, and new technology, the scope of projects is able to expand into new use cases we have yet to see come online. Opportunities for gaming, exchanges, and applications yet to be created will be able to come online with better user experience.",
```

Comment:
Good point but feels a bit wordy. I think we could make this point more concise.